Source/Atomic/UI/UIListView.h

```diff
@@ -98,6 +98,10 @@ class UIListView : public UIWidget
 
     void SelectItem(ListViewItem* item, bool select);
 
+    ListViewItem* pivot;
```

Comment:
member variables should be post-pended with an underscore, so pivot_, pivotIndex_, startNewSelection_